addOns/pscanrules/CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - The following scan rules now have alert references (Issue 7100):
     - Cookie without SameSite Attribute 
     - Retrieved from Cache (raw text was also trimmed from one Alert reference (Issue 8262))
+    - View State Scan Rules (Issue 8262)
```

Comment:
Needs to be moved to Unreleased section.

Reply:
ok